tools/specs/Microsoft.Data.SqlClient.nuspec

```diff
@@ -33,6 +33,7 @@ When using NuGet 3.x this package requires at least version 3.4.</description>
         <dependency id="Microsoft.Identity.Client" version="4.22.0" />
         <dependency id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="6.8.0" />
         <dependency id="Microsoft.IdentityModel.JsonWebTokens" version="6.8.0" />
+        <dependency id="System.Configuration.ConfigurationManager" version="4.7.0" exclude="Compile" />
```

Comment:
If I am not mistaken Previously we had 5.0.0 as the version in version props. Why did we downgraded the version here?

Reply:
It was always 4.7.0 in Nuspec for other frameworks, 5.0.0 was overlapping it - upgrade is not intended for next release.